packages/cozy-harvest-lib/package.json

```diff
@@ -48,6 +50,13 @@
     "@babel/core": "7.16.12",
     "@babel/register": "7.16.9",
     "@cozy/cli-tree": "^0.7.0",
+    "@storybook/addon-essentials": "^7.0.23",
```

Comment:
All the storybook dependencies should be fixed versions to comply with Cozy standards